Tests/Util/FilenameUtilsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Vich\UploaderBundle\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+use Vich\UploaderBundle\Util\FilenameUtils;
+
+class FilenameUtilsTest extends TestCase
+{
+    /**
+     * @dataProvider spitNameByExtensionProvider
+     */
+    public function spitNameByExtension($filename, $basename, $extension)
+    {
+        $this->assertSame([$basename, $extension], FilenameUtils::spitNameByExtension($filename));
+    }
+
+    public function spitNameByExtensionProvider()
+    {
+        return [
+            ['Текстовый файл.txt', 'Текстовый файл', 'txt'],
```

Comment:
Still I can't easily understand what is done in each of the elements of this array, since key are (implicit) numeric. If you replace them with string keys, it can be easier to understand. For example, the first element could have a key like 'filename with extension', the second element 'cyrillic filename with whitespace' (just a wild guess), and so on